locust/static/locust.js

```diff
@@ -102,7 +102,7 @@ $(".stats_label").click(function(event) {
     alternate = false;
     totalRow = report.stats.pop()
```

Comment:
Need to fix this since totalRow is no longer at the end. I have it locally; waiting to test and then will update the PR.